### PR TITLE
[tests-only][full-ci] add tests to create quick link of space drive using root endpoint

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR add tests coverage to create quick link share of a project-space drive using root endpoint.
``` feature
Scenario Outline: create quick link share of a project space drive using root endpoint
Scenario: create an internal quick link share of a project space drive using root endpoint
```
## Related Issue

- Related to : https://github.com/owncloud/ocis/issues/9046

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
LGTM. please, rebase the PR

**Review by saw-jan — APPROVED** [review]

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature [hunk 1]
```diff
@@ -3817,4 +3817,129 @@ Feature: Create a share link for a resource
...
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role |
+      | view             |
+      | upload           |
+      | edit             |
+      | createOnly       |
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3886]: can we add secure view here ?

Comment (prashant-gurung899) [reply tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3886]: I think there's no secure view option while creating quick link share. @amrita-shrestha

Comment (S-Panta) [reply tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3886]: Currently, no scenerio should be added for Secure View for sharing project drive.

**Comment by S-Panta** [comment]
@nirajacharya2  There are some redundant changes in this file. Prob due to some linter.

File: tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature (hunk 1 above)
Comment (amrita-shrestha) [reply tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3886]: @prashant-gurung899 better if you add such information in the description. Important details should be mentioned in the description

Comment (prashant-gurung899) [reply tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature:3886]: Sure!

**Comment by prashant-gurung899** [comment]
> @prashant-gurung899 There are some redundant changes in this file. Prob due to some linter.
@S-Panta I've used gherlint to fix lint issues on feature files.

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Review by saw-jan — APPROVED** [review]

**Pull request merged**